src/Jackett.Common/Indexers/BitHdtv.cs

```diff
@@ -123,95 +115,96 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
             var releases = new List<ReleaseInfo>();
-
             var qc = new NameValueCollection();
-            if (!string.IsNullOrWhiteSpace(query.ImdbID))
+            var results = new List<WebClientStringResult>();
+            var search = new UriBuilder(SearchUrl);
+            if (query.IsImdbQuery)
             {
                 qc.Add("search", query.ImdbID);
-                qc.Add("options", "4");
+                qc.Add("options", "4"); //Search URL field for IMDB link
+                search.Query = qc.GetQueryString();
+                results.Add(await RequestStringWithCookiesAndRetry(search.ToString()));
```

Comment:
I like more the old school method `SearchUrl + qc.GetQueryString()` but up to you.
I tested the tracker and it's working.